### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by BaBoWaGa** [3liz/lizmap-web-client#3671]
### What is the bug?

With the new version of Lizmap you can use opentopomap as base layer, but it doesn't appear in the base layer list when you open the map on the web

### Steps to reproduce the issue

![image](https://user-images.githubusercontent.com/109667824/236595122-22945349-3912-481d-827a-88916d12de67.png)

### Versions

**Versions :**

* Lizmap Web Client : 3.6.3
* Lizmap plugin : 3.13.0
* QGIS Desktop : 3.28.5
* QGIS Server : 3.30.2
* Py-QGIS-Server : not used
* QGIS Server plugin atlasprint : 3.3.2
* QGIS Server plugin lizmap_server : 2.7.1
* QGIS Server plugin wfsOutputExtension : 1.7.1

### Check Lizmap plugin

- [X] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

Ubuuntu 22

### Browsers

Firefox, Chrome, Microsoft Edge

### Browsers version

chrome 112

**Comment by jeremieprudhomme** [comment]
I have the same behavior. OpenTopoMap checked in the Lizmap plugin, not appearing in the basemap dropmenu.
Versions :
    Lizmap Web Client : 3.6.3
    Lizmap plugin : 3.9.10
    QGIS Desktop : 3.22.16
    QGIS Server : 3.22.16

**Comment by aeduard** [comment]
Same behavior also here with the newest updates as of today.

Lizmap Web Client : 3.6.4
Lizmap plugin : 3.14.0
QGIS Desktop : 3.28.6
QGIS Server : 3.28.6

**Comment by aeduard** [comment]
I've updated lizmap but OpenTopoMap is still not appearing in the Base layer drop-down... I've tried with two existing maps.

Lizmap Web Client : 3.6.5
Lizmap plugin : 3.15.1
Lizmap QGIS Server plugin 2.8.0
QGIS Desktop : 3.28.6
QGIS Server : 3.28.6

**Comment by Gustry** [comment]
Indeed.
The plugin is generating a true boolean value, while the code in LWC is expecting a string.
Update your QGIS plugin desktop plugin later this week, it will be OK.

Note, this feature is deprecated for Lizmap Web Client 3.7, there won't be any checkbox needed and you will be free to use whatever background you want.

**Comment by aeduard** [comment]
I've updated the desktop plug-in, and resaved the map. Now the OpenTopoMap is appearing in the dropdown, but no base map is loaded, just blank/white background.

**Comment by Gustry** [comment]
Look at your request in the webbrowser console, you should see request going to the opentopomap.org ?

**Comment by aeduard** [comment]
Yes, there are multiple requests, but all fails with Cross-Origin Request Blocked:... see the attached screenshot

![opentopomap_error](https://github.com/3liz/lizmap-web-client/assets/16295364/33896ca9-050c-4860-a30e-40bffe723cd6)
